daemon/mgr/container.go

```diff
@@ -351,7 +351,7 @@ func (mgr *ContainerManager) Start(ctx context.Context, id, detachKeys string) (
 	})
 	if err == nil {
 		c.meta.State.Status = types.StatusRunning
-		c.meta.State.StartedAt = time.Now().String()
+		c.meta.State.StartedAt = time.Now().UTC().Format(utils.TimeLayout)
```

Comment:
We can't use `StartedAt` for `CreatedAt`, right? Because `startedAt` is used for running, not `created`. 
 
I think we should create `createdAt` field for api.

Reply:
I think so. And maybe we should  add `StartedAt` in `types.Container{}` and add `CreatedAt` in `types.ContainerState{}` .  That means we can describe the running time by Status like this:
```
$ pouch ps
Name     ID       Status         Image                              Runtime   Created
1dad17   1dad17   Up 1 minute    docker.io/library/busybox:latest   runv      1 hour ago
505571   505571   Up 5 seconds   docker.io/library/busybox:latest   runc      1 hour ago
```
I can add this feature in a follow-up PR. 😄  WDYT @0x04C2

Reply:
Agree! @allencloud